examples/multisig_plain/lib.rs

```diff
@@ -0,0 +1,837 @@
+        /// This also confirms the transaction for the caller.
+        /// This can be called by any owner.
+        #[ink(message)]
+        fn submit_transaction(&mut self, transaction: Transaction) {
+            self.ensure_caller_is_owner();
+            let trans_id = self.transactions.put(transaction);
+            self.confirmation_count.insert(trans_id, 0);
+            self.env().emit_event(Submission {
+                transaction: trans_id,
+            });
+            self.confirm_by_caller(self.env().caller(), trans_id);
+        }
+
+        /// Remove a transaction from the contract.
+        /// Only callable by the wallet itself.
+        /// Panics if `trans_id` is no valid transaction id.
+        #[ink(message)]
+        fn cancel_transaction(&mut self, trans_id: TransactionId) {
+            self.ensure_from_wallet();
+            if self.take_transaction(trans_id).is_some() {
+                self.env().emit_event(Cancelation {
+                    transaction: trans_id,
+                });
+            }
+        }
+
+        /// Confirm a transaction for the sender that was submitted by any owner.
+        /// This can be called by any owner.
+        /// Panics if `trans_id` is no valid transaction id.
+        #[ink(message)]
+        fn confirm_transaction(&mut self, trans_id: TransactionId) {
+            self.ensure_caller_is_owner();
+            self.ensure_transaction_exists(trans_id);
+            self.confirm_by_caller(self.env().caller(), trans_id);
+        }
+
+        /// Revoke the senders confirmation.
+        /// This can be called by any owner.
+        /// Panics if `trans_id` is no valid transaction id.
+        #[ink(message)]
+        fn revoke_confirmation(&mut self, trans_id: TransactionId) {
+            self.ensure_caller_is_owner();
+            let caller = self.env().caller();
+            if self.confirmations.remove(&(trans_id, caller)).is_some() {
+                mutate_map(&mut self.confirmation_count, &trans_id, |count| *count -= 1);
+                self.env().emit_event(Revokation {
+                    transaction: trans_id,
+                    from: caller,
+                });
+            }
+        }
+
+        /// Execute a confirmed execution.
+        /// Its return type indicates whether the called transaction was succesful.
+        /// This can be called by anyone.
+        #[ink(message)]
+        fn execute_transaction(&mut self, trans_id: TransactionId) -> Result<(), ()> {
+            self.ensure_confirmed(trans_id);
+            let t = self.take_transaction(trans_id).expect(WRONG_TRANSACTION_ID);
+            let result = env::call::CallParams::<EnvTypes, ()>::invoke(
+                t.callee,
+                t.selector.into(),
+            )
+            .gas_limit(t.gas_limit)
+            .transferred_value(t.transferred_value)
+            .push_arg(&CallInput(&t.input))
+            .fire()
+            .map(|_| ())
+            .map_err(|_| ());
```

Comment:
Have a hard time to believe that this is what `rustfmt` actually formatted this code to. 0.o

Reply:
It did and the CI agrees that it's OK.